src/pages/Resume.jsx

```diff
@@ -64,40 +61,175 @@ const Resume = () => {
+                            provided valuable support to Computer Science and
+                            Mathematics faculty members by facilitating student
+                            learning through individual and group tutoring
+                            sessions, as well as assessing their academic
+                            progress by grading papers. This role not only
+                            allowed me to hone my teaching and mentoring skills
+                            but also enabled me to make meaningful contributions
+                            towards the academic development of the university's
+                            students.
                         </code>
                     </div>
                 </div>
-                <div className="flex flex-row pt-10 flex-wrap">
+                <div className="flex flex-col pt-10 flex-wrap md:flex-row">
                     <div className="w-1/4">
                         <code className="text-yellow_vs text-2xl">Skills</code>
                     </div>
-                    <div className="w-3/4">
-                        <code className="text-m">
-                            <br />• Nulla tristique luctus lacinia. Nullam ut
-                            tortor arcu.
-                            <br />• Aenean in mi non leo placerat suscipit a
-                            eget odio.
-                            <br />• Vivamus et dolor odio.
-                            <br />• Suspendisse tempus interdum eros, et
-                            bibendum massa tempus vitae.
-                            <br />• Sed in aliquam ligula. Sed eget orci tortor.
-                            <br />• Pellentesque laoreet laoreet justo, sed
-                            efficitur sapien tincidunt eu.
-                            <br />• Nulla tempor nunc eu molestie volutpat. Nunc
-                            a accumsan dolor.
-                            <br />• Duis ut congue dui.
-                        </code>
+                    <div className="md:w-3/4">
+                        <p className="text-m font-mono">
+                            <br />• I speak Latin :)
+                            <br />• Lorem ipsum dolor sit amet, consectetur
+                            adipiscing elit, sed do eiusmod tempor incididunt ut
+                            labore et dolore magna aliqua. Ut enim ad minim
+                            veniam, quis nostrud exercitation ullamco laboris
+                            nisi ut aliquip ex ea commodo consequat. Duis aute
+                            irure dolor in reprehenderit in voluptate velit esse
+                            cillum dolore eu fugiat nulla pariatur. Excepteur
+                            sint occaecat cupidatat non proident, sunt in culpa
+                            qui officia deserunt mollit anim id est laborum.
+                            <br />
```

Comment:
you can speak LATIN?!?!? What does this say then?

Reply:
"Once upon a time, in a land of endless meetings and broken code, a developer desperately tried to fix a bug. After hours of frustration and too much coffee, they realized the issue was just a missing semicolon. They facepalmed, fixed it, and swore to never speak of it again. The end."